frontend/docker-entrypoint.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 export SLURM_CPUS_ON_NODE=$(cat /proc/cpuinfo | grep processor | wc -l)
-sudo sed -i "s/REPLACE_IT/CPUs=${SLURM_CPUS_ON_NODE}/g" /etc/slurm-llnl/slurm.conf
+sudo sed -i "s/REPLACE_IT/${SLURM_CPUS_ON_NODE}/g" /etc/slurm/slurm.conf
 
 sudo service munge start
 sudo service ssh start
```

Comment:
```suggestion
sudo slurmctld
```

Reply:
The different containers need different daemons launched.  The new version launches the slurm daemons directly intead of via `service ... start`.  The frontend doesn't need slurmctld, that only goes on the master.  All containers need the ssh service, however.